### ISSUE TITLE
Failed mux-player-react Firefox accessibility test

### ISSUE THREAD

**Issue by karlodr** [muxinc/elements#1040]
**Description:**
I’ve been trying to pass an accessibility test in Firefox but haven’t been successful despite attempting several options.

**Issue:**
![image](https://github.com/user-attachments/assets/afc67bf7-5807-4252-b4eb-f54efcda89ac)

Is there a recommended way to address this issue and pass the test?

**Steps to Reproduce:**
1. Use `mux-player-react` component
2. Open the app via firefox
3. Navigate to accessibility tools; it is usually inside the `region` role
4. The error should be visible (Check the image above)

**Expected Behavior:**
The error should not occur, or there should be a way to handle it via props.

**Actual Behavior:**
No props are available to address this, resulting in an accessibility error.

**Comment by anki247** [comment]
+1
I also see 2 accessibility issue report in Google PageSpeed Insight 

<img width="1784" height="932" alt="Image" src="https://github.com/user-attachments/assets/c83ae716-01b4-4a03-9fb2-f7c428b644e0" />

**Comment by Pintorado** [comment]
Hi @karlodr @anki247, thanks for reporting this. I will look into it shortly and get back to you when I have something

**Comment by Pintorado** [comment]
Hi again, I’ve implemented several accessibility enhancements in media-chrome’s components that should resolve this firefox accessibility issue. I’ll let you know when we release a new version with this enhancements so you can update `mux-player-react` with the fix
